mlrun/__main__.py

```diff
@@ -932,6 +938,7 @@ def project(
     timeout,
     ensure_project,
     schedule,
+    overwrite_schedule,
```

Comment:
I like type hints (`overwrite_schedule: bool`) but I see there are no hints to be found here... So I guess its fine.

Reply:
Ok, remains unchanged